assets/js/main.js

```diff
@@ -0,0 +1,58 @@
+$(document).ready(function() {
```

Comment:
Why use jquery here for interaction? Wouldn't it be better to just use plain javascript instead? Just a suggestion.

Reply:
If it is possible using vanilla JavaScript, it would be better. I am not sure if @GusBaamonde is enough familiar with JavaScript for that.

If not, we could add a task on the board for doing that later.

Anyway, the usage of `.js` seems to be not required right? I mean, in the webapp we could implement the same trivial logic with react. Right?

Reply:
If you don't know how to go on with this we can discuss it.

Reply:
I think this is to showcase how the styles work in this small proyect. Right @GusBaamonde ? The behaviour in webapp should be made in react.

Reply:
Yes @cbernat is to see how it works